server.js

```diff
@@ -64,14 +66,31 @@ app.get("/api/tour/:tour", async (req, res) => {
   res.sendFile(path.join(__dirname, "client/build", "index.html"));
 });
 
+//ImageRoutes
+app.get("/api/locationImages/:locationName", async (req, res) => {
+  const { locationName } = req.query;
+  try {
+    const locationValue = await getImageDataOfLocation(locationName);
+    if (!locationValue) {
+      res.status(404).send("could not find the content you are looking for");
```

Comment:
```suggestion
      res.status(404).send("Could not find the content you are looking for");
```

Reply:
😄  just saw your comments now. fixed a lot of things here and there